docs/src/route.ts

```diff
@@ -4,18 +4,19 @@ const ROUTES = {
   productAdvanced: '/branding/x-advanced/',
   productTemplates: '/branding/templates/',
   productDesignKits: '/branding/design-kits/',
+  newCareers: '/branding/careers',
+  pricing: '/branding/pricing/',
+  about: '/branding/about/',
   materialIcons: '/components/material-icons/',
   freeTemplates: '/getting-started/templates/',
   components: '/getting-started/supported-components/',
   customization: '/customization/how-to-customize/',
   theming: '/customization/theming/',
   documentation: '/getting-started/usage/',
-  pricing: '/branding/pricing/',
   blog: '/blog/',
   showcase: '/discover-more/showcase',
   roadmap: '/discover-more/roadmap',
   languages: '/discover-more/languages',
-  about: '/branding/about/',
   vision: '/discover-more/vision/',
   careers: '/company/careers/',
```

Comment:
I figure this `careers` is the route where the `See open roles` button on the about page takes it to. We should change it to link to this new one!